docs/spec/sources.md

```diff
@@ -0,0 +1,147 @@
+# Sources
+
+A **source** is any resource that generates or imports an event and relays that
+event to another resource on the cluster via CloudEvents. Sourcing events is
+critical to developing a distributed system that reacts to events.
+
+A Source:
+
+- Represent an off or on-cluster system, service or application.
```

Comment:
`Represents` ?

Reply:
I am not sure if the _source_ itself does represent that system, service or app.

I usually tell people that with a source, you are able to connect to a 3rd party system, service or app - living on the cluster or somewhere outside.

Reply:
you are right, it should be `Represents the request for events from...`